Gemfile

```diff
@@ -67,3 +67,11 @@ gem 'stripe'
 gem 'stripe_event'
 
 gem 'droplet_kit'
+
+#added specific gems for Windows platform. Switch to JRuby, which would have the same platform string across both Windows and Linux 
+
+gem 'net-ssh' '2.10.0', :platforms => :jruby
```

Comment:
NOTE: If you are running on jruby you need to install jruby-pageant manually (gemspec doesn't allow for platform specific dependencies)